Topics/Development_Process/Conda/Setup_Conda.md

```diff
@@ -0,0 +1,156 @@
+## What is Conda and why do you need it? 
```

Comment:
You probably want to introduce the other alternatives to Conda, because there are also virtualenv and pipenv, which you want to compare the strengths and weaknesses of each